OrbitGl/CMakeLists.txt

```diff
@@ -152,7 +152,10 @@ target_sources(OrbitGlTests PRIVATE
                ScopedStatusTest.cpp
                SliderTest.cpp
                TimerInfosIteratorTest.cpp
-               UnitTestMockEnvironment.cpp)
+               # Todo (176066030): OrbitGl relies on flags usually defined in the OrbitQt executable, so the
+               # tests needs this to link against them. To avoid multiple definitions, we compile the flags
+               # cpp with the tests
+               ../OrbitQt/ClientFlags.cpp)
```

Comment:
Ah good call to factor them out, thanks. Could they be moved to OrbitGl however so that OrbitGl wouldn't depend on OrbitQt?

Reply:
I was thinking the same, but then again, some of the flags are defined in OrbitQt, so it again would be rather inconsistent to define them in OrbitGl, wouldn't it?

But probably from a dependency perspective it's still more consistent, I'll change it.

Reply:
Changed

Reply:
Thanks!